admin/base/settings/defaults.py

```diff
@@ -219,3 +224,10 @@
 
 ENTRY_POINTS = {'osf4m': 'osf4m', 'prereg_challenge_campaign': 'prereg',
                 'institution_campaign': 'institution'}
+
+
+# Change desk settings in local.py
+DESK_KEY = ''
+DESK_KEY_SECRET = ''
+DESK_TOKEN = ''
```

Comment:
I don't see this as worthy of its own PR. I have already gone back and forth a few times between different options so changing it one more time shouldn't take too long...